packages/console/src/Console.jsx

```diff
@@ -825,6 +810,8 @@ export class Console extends PureComponent {
     } = this.state;
     const consoleMenuObjects = this.getObjects(objectMap);
     const inputMaxHeight = Math.round(consoleHeight * 0.7);
+    const isDisconnected = disconnectedChildren != null;
```

Comment:
Nit: This is only used once, though it does make the JSX more readable.

Reply:
What do you mean it's only used once? `isDisconnected` is used in a few places, I figured it was easier than putting `disconnectedChildren != null` everywhere.

Reply:
What do you mean it's only used once? `isDisconnected` is used in a few places, I figured it was easier than putting `disconnectedChildren != null` everywhere.

Reply:
Ah I see. Sorry was looking at the new additions only...ignore this.